### PR TITLE
Added Compound Interest Java Code

### PR DESCRIPTION
**Pull request by Aman28801**
**Issue:**

### Description
I added the code for compound interest in java..   Issue #201 
This is my contribution towards hacktoberfest so please review it and give the hacktoberfest aceepted label 
and if u need any changes please let me know i improve it as soon as possible

### CODE REVIEW

File: src/pages/programs/calculate-compound-interest.mdx
Comment (uiuxarghya) [inline src/pages/programs/calculate-compound-interest.mdx:1]: The content of this mdx file is not written properly.  Check this file [Add Two Integers](https://github.com/javaistic/javaistic/blob/main/src/pages/programs/add-two-integers.mdx)
You need to follow this format while writing the mdx pages.

**Review by uiuxarghya — CHANGES_REQUESTED** [review]

File: src/pages/programs/calculate-compound-interest.mdx
Comment (Aman28801) [reply src/pages/programs/calculate-compound-interest.mdx:1]: okay i will update the file and update it in 1 hr...thanks

Comment (uiuxarghya) [reply src/pages/programs/calculate-compound-interest.mdx:1]: okay

Comment (Aman28801) [reply src/pages/programs/calculate-compound-interest.mdx:1]: Done please check it now

Comment (uiuxarghya) [reply src/pages/programs/calculate-compound-interest.mdx:1]: i have updated the file.

**Review by uiuxarghya — APPROVED** [review]

**Comment by uiuxarghya** [comment]
Thanks @Aman28801 for the contribution. 💯
Consider adding a star ⭐ to the repo.

**Pull request merged**